src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -53,15 +53,15 @@ const AnonymousChannelListScreen = ({route}) => {
     <>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+        <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Search` component now receives a new prop `isAnon` set to `true` and the `onPress` function has been updated to call `goToContactScreen` with `{from: ANONYMOUS}` as argument. This change seems fine as long as these props are properly handled in the `Search` component.
```diff
- <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+ <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->